src/core/Memory.test.ts

```diff
@@ -1,12 +1,23 @@
-import {getValues, saveValues} from "./Memory";
+import {getValues, saveValues, storageKey} from "./Memory";
+
+beforeEach(() => {
+    localStorage.clear();
+})
 
 test("Test That Value Retrieved will Be Empty If not Set", () => {
     expect(getValues()).toEqual([]);
 })
 
-test("Test that Value Can Be Saved And Retrieved", () => {
+test('Test That Value Will Be Saved In LocalStorage', () => {
     const value = [{date: new Date(), value: 21}, {date: new Date(), value: 48}];
```

Comment:
Do you think that we should introduce constant if something happens twice in tests?

Reply:
shure, shame :)